projects/testing-library/schematics/ng-add/schema.json

```
@@ -3,6 +3,18 @@
   "$id": "SchematicsTestingLibraryAngular",
   "title": "testing-library-angular",
   "type": "object",
-  "properties": {},
+  "properties": {
+    "installJestDom": {
+      "type": "boolean",
+      "description": "Would you like to install jest-dom?",
```

Could you also add a `x-prompt` here please.

```suggestion
      "description": "Install jest-dom as a dependency.",
      "x-prompt": "Would you like to install jest-dom?"
```